hail/python/hail/docs/conf.py

```diff
@@ -20,6 +20,10 @@
 import os
 import sys
 
+# sphinx essentially evals this file, so we cannot rely on package structure
```

Comment:
this seems hacky. What is wrong with environment variables?

I think we have the same dependency structure using env variables or the generated version info file.